websites/J/jpdb/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "988699263775154176",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let dueCountElem: Element, deckTitle: string, query: string, meaning: string;
```

Comment:
none of these should be declared globally as you are not gaining anything from doing that. Please remove them

Reply:
should `decklist` be in the check instead too?
```ts
if ([
      "/novel/",
      "/vocabulary-list/",
      "/textbook/",
      "/anime/",
      "/youtube-video/",
      "/live-action/",
      "/video-game/",
      "/web-novel/",
      "/visual-novel/",
      "/audio/",
  ].some(deck => document.location.pathname.includes(deck))
}
```

Reply:
you can keep that one as a global constant. I'm only saying you don't need the variables